### PR TITLE
remove single child statements

### PR DESCRIPTION
**Pull request by mr-tz**
see #382 and https://github.com/fireeye/capa/pull/563

### CODE REVIEW

File: host-interaction/os/version/check-os-version.yml [hunk 1]
```diff
@@ -23,21 +23,21 @@ rule:
       - mnemonic: cmp
       - or:
         - and:
-          - number: 5 # Windows 2000
+          - number: 5 = Windows 2000
```
Comment (mr-tz) [inline host-interaction/os/version/check-os-version.yml:26]: minor update that slipped in here

**Comment by mr-tz** [comment]
fail on `check OS version` (https://github.com/fireeye/capa-rules/pull/386/checks?check_run_id=2619626441#step:9:22) due to https://github.com/fireeye/capa/issues/564

File: host-interaction/os/version/check-os-version.yml [hunk 2]
```diff
@@ -10,7 +10,6 @@ rule:
       - Discovery::System Information Discovery [T1082]
     examples:
       - 493167E85E45363D09495D0841C30648:0x401000
-      - 477743976643213d96b66ed5041a3b12:0x43CFF6
```
Comment (mr-tz) [inline host-interaction/os/version/check-os-version.yml:13]: removing this library function as recognized by FLIRT

**Comment by williballenthin** [comment]
test fixed here: https://github.com/fireeye/capa-rules/pull/387

**Comment by williballenthin** [comment]
@mr-tz  ok cool do the test fixup here and i'll close the other PR

**Comment by mr-tz** [comment]
knock knock
race condition
who's there?

**Pull request merged**